sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="System.Memory" Version="4.5.2" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Azure.Core\Azure.Core.csproj">
+    <ProjectReference Include="..\src\Azure.Core.csproj">
       <Project>{FEC0AE57-E241-46DB-B7C7-9181F28E1D3F}</Project>
       <Name>Azure.Core</Name>
     </ProjectReference>
```

Comment:
@maririos at some point we will want to move the samples up a level into a samples directory parallel to the tests directory.

Reply:
Ohh interesting. Is it part of the repo structure guidelines and we just missed it?

Reply:
Yes see. https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/repo-structure.md we hope to make the samples individually runnable separate from our tests. We also hope to try and tie them into the sample code in our README's for the libraries.

Reply:
I see. Issue https://github.com/Azure/azure-sdk-for-net/issues/6085 created to address that.

As for the Readme, we do add code snippets from the Hello World sample, following the [Reame Guidelines](https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md)